fw-child/resources/js/child-functions.js

```diff
@@ -609,47 +609,77 @@ const $ = jQuery;
     
       $('.scroll-card').each(function () {
         const card = this;
-    
+        const isLastCard = $(card).closest('.fw-query-item').is(':last-child');
+
```

Comment:
Maybe it's already good, but could you check if the empty lines (here and elsewhere) have any whitespace ? If they have, I would remove them (but keep the empty line). Only check the new code in this commit, not the whole file.